vivisect/qt/views.py

```diff
@@ -444,3 +571,32 @@ def vivAddName(self, nifo):
         else:
             self.vivSetData(va, 1, name)
 
+
+#FIXME: is this a good time to use a @decorator?
```

Comment:
You could introduce a wrapper that sets an instance attribute on every instance of these classes (not sure how you'd do it at the class attribute level), but you'd have to wrap the every instance of the class you want, so it adds a layer of indirection, and the actual wrapping introduces more complexity.

I'd say pass for now and remove the FIXME. If we every have to add more of these, it might be worth adding, but right now it's more complexity than what is necessary, and adds a fair bit of overhead.